Doc/library/string.rst

```diff
@@ -257,7 +257,7 @@ on the value, ``'!r'`` which calls :func:`repr` and ``'!a'`` which calls
 Some examples::
 
    "Harold's a clever {0!s}"        # Calls str() on the argument first
-   "Bring out the holy {name!r}"    # Calls repr() on the argument first
+   "Bring out the holy {name!r}"    # Calls repr() on the keyword argument 'name'
```

Comment:
I think this is not a typo but a miscomprehension.  The previous line doesn’t mean «calls str on the first argument», but (if I understand right) «calls str on the argument first, before substituting the result in the formatted string».